.github/actions/breaking-pr-check/index.js

```diff
@@ -1,12 +1,17 @@
+// @ts-check
+/* eslint-disable jsdoc/no-types */
```

Comment:
This makes me think we should change `eslint.config.mjs` to only apply the TypeScript rules to TypeScript files. That's fine as a followup IMO.